frontend/ui/detailgraph.jsx

```diff
@@ -1,12 +1,18 @@
-import _ from 'lodash';
 import React from 'react';
+import _ from 'lodash';
 import Dimensions from 'react-dimensions';
 import { curveLinear } from 'd3';
 import { timeFormat } from 'd3-time-format';
 import MetricsGraphics from 'react-metrics-graphics';
 
 
-class MeasureGraph extends React.Component {
+class DetailGraph extends React.Component {
+  shouldComponentUpdate(nextProps) {
+    return !_.isEqual(this.props.seriesList, nextProps.seriesList) ||
```

Comment:
This is quite noticeable when changing the percentiles and the normalize checkbox. Well solved!

Reply:
Yeah, I'm pretty happy with how easy this was!